chart/crds/secrets.hashicorp.com_hcpvaultsecretsapps.yaml

```diff
@@ -169,6 +176,8 @@ spec:
                           - name
                           type: object
                         type: array
+                    required:
```

Comment:
The behaviour is fine as it has a default, but from a generated documentation perspective this seems a little odd to be labelled as required. If you add `,omitempty` to the JSON tag will it stop labelling it as required?

Reply:
Yes, it would be not be included in the required field set. I think we set the default to false to avoid some perpetual diff scenarios with Rancher.